lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -958,24 +950,6 @@
         }
       }
 
-      .actions-secondary {
```

Comment:
@clrux  - Making sure this isn't something we use for one of our message states would be a good check. I'm not sure offhand why this section would need to go away.